src/main/java/seedu/address/storage/JsonAdaptedIssue.java

```diff
@@ -57,15 +56,15 @@ public JsonAdaptedIssue(Issue source) {
         deadline = source.getDeadline().toString();
         status = source.getStatus().toString();
         issueId = source.getIssueId().toString();
-        project = new JsonAdaptedProject(source.getProject());
+        project = source.getProject().getId().toString();
```

Comment:
this is projectId? since its getting the Id

Reply:
Changed to `getProjectId()` to add clarity

Reply:
As in the variable name projectId instead of project? sorry

Reply:
Sorry, as in the variable name being projectId instead of project?